Add: 물품 등록 Page

hyeonjin6530 opened the pull request:
<물품등록 page Route 추가>
       modified:   src/App.js

<물품 등록 page 개발>
       new file:   src/pages/GoodsRegistration.js

<navigate 경로 변경: 기존(/) -> 변경후(/main)>
       modified:   src/pages/UserRental.js

Commit:
Add: 물품 등록 Page
<물품등록 page Route 추가>
       modified:   src/App.js

<물품 등록 page 개발>
       new file:   src/pages/GoodsRegistration.js

<navigate 경로 변경: 기존(/) -> 변경후(/main)>
       modified:   src/pages/UserRental.js

VarGun reviewed (commented):
## GoodsRegistration.js
- ImageContainer
    - padding-bottom : 자신이 포함하고 있는 요소가 아닌 다른 외부 요소와의 간격이기 때문에 margin 을 사용해야 함.
- 물품 등록 Button 눌렀을 때, image와 각 input 값들이 들어가 있는지 체크하고 아니라면 해당 값에 대해 alert 메시지를 띄워주세요.
- **Input Component 를 develop 브랜치에 수정해두었으니 pull 받고 진행해주세요.**
- Check 값이 변하는 것을 멘트( 반납이 필요한 물품입니다. )랑 같이 묶여있는 박스 전체에 적용해주세요.
    - 멘트 부분을 클릭해도 Check 상태가 변하도록

hyeonjin6530 committed:
Fix: 물품 등록 Page

Merge branch 'develop' of https://github.com/2023-2-WINK-Project-Team1/front-end into front-end-42
count.js (상태관리를 위해 value, setValue 추가)
GoodsRegistration.js 수정
